pandapower/create.py

```diff
@@ -2758,6 +3451,84 @@ def create_switch(net, bus, element, et, closed=True, type=None, name=None, inde
     return index
 
 
+def create_switches(net, buses, elements, et, closed=True, type=None, name=None, index=None, z_ohm=0, **kwargs):
+    """
+    Adds a switch in the net["switch"] table.
+
+    Switches can be either between two buses (bus-bus switch) or at the end of a line or transformer
+    element (bus-element switch).
+
+    Two buses that are connected through a closed bus-bus switches are fused in the power flow if
+    the switch is closed or separated if the switch is open.
+
+    An element that is connected to a bus through a bus-element switch is connected to the bus
+    if the switch is closed or disconnected if the switch is open.
+
+    INPUT:
+        **net** (pandapowerNet) - The net within which this switch should be created
+
+        **buses** (list)- The bus that the switch is connected to
+
+        **element** (list)- index of the element: bus id if et == "b", line id if et == "l", trafo id if \
+            et == "t"
+
+        **et** - (list) element type: "l" = switch between bus and line, "t" = switch between
+            bus and transformer, "t3" = switch between bus and transformer3w, "b" = switch between
+            two buses
+
+    OPTIONAL:
+        **closed** (boolean, True) - switch position: False = open, True = closed
+
+        **type** (int, None) - indicates the type of switch: "LS" = Load Switch, "CB" = \
+            Circuit Breaker, "LBS" = Load Break Switch or "DS" = Disconnecting Switch
+
+        **z_ohm** (float, 0) - indicates the resistance of the switch, which has effect only on
+            bus-bus switches, if sets to 0, the buses will be fused like before, if larger than
+            0 a branch will be created for the switch which has also effects on the bus mapping
+
+        **name** (string, default None) - The name for this switch
+
+    OUTPUT:
+        **sid** - The unique switch_id of the created switch
+
+    EXAMPLE:
+        create_switch(net, bus =  0, element = 1, et = 'b', type ="LS", z_ohm = 0.1)
+
+        create_switch(net, bus = 0, element = 1, et = 'l')
+
+    """
+
+    nr_switches = len(buses)
```

Comment:
This still needs to be changed: net.switch.index instead of net.line.index. We also need to perform the same checks as in create_switch (checking in the line and trafo tables) and make sure that for all new created switches everything is in order.

Reply:
Now should be correct